content/influxdb/v2/process-data/common-tasks/downsample/_index.md

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
This doc shouldn't be modified in the v2 docs. This sections is specifically focused on Flux tasks (scheduled Flux queries).